src/litdata/processing/data_processor.py

```diff
@@ -902,7 +902,7 @@ def _upload_index(self, output_dir: Dir, cache_dir: str, num_nodes: int, node_ra
             self._upload_index(output_dir, cache_dir, 1, None)
 
 
-class DataTransformRecipe(DataRecipe):
+class MapRecipe(DataRecipe):
```

Comment:
could people be relying on those names? Is it worth to put in a backwards compatible alias?

Reply:
No, this is internal